TMSFTT/urls.py

```diff
@@ -19,6 +19,7 @@
 
 
 API_URLPATTERNS = [
+    path('training_record', include('training_record.urls')),
```

Comment:
```suggestion
    path('training-record', include('training_record.urls')),
```
As suggested in https://github.com/DLUT-SIE/TMSFTT-BE/pull/9/files#r248498944